content/blog/kallisti5/2021-03-22_feb_2021_survey_results.md

```diff
@@ -0,0 +1,138 @@
+* "I'm scared to run it on real hardware in case I lose data."
+* "Lack of a programmers editor that fits my workflow"
+* "I really want to use Haiku on Raspberry Pi 400."
+* "I don't use haiku because it doesn't support my WiFi card driver."
+
+### If one sentence could summarize the most difficult task you have experienced in Haiku, what would it be?
+
+* "playing hi res video in media player."
+* "Generally, Haiku is easy to use, but the installer only works for people which are into tech, not so for normal users."
+* "Modern browsing. It's a pain!"
+* "Installing Haiku on a UEFI system"
+* "Getting old BeOS applications to run."
+* "Trying to build GCC 10.2.0 in Haiku."
+* "Trying to blacklist or replace a driver"
+* "Figuring out how to get default video to start properly. "
+
+## Any other feedback?
+
+**Here are a few of my favorite positive messages :-)**<br/>
+
+* "Besides the browser issues, I must admit that WebPositive had improved a lot in the recent times.  Several sites that didn't worked 6 months ago, now works."
+* "Keep up great work! I love Haiku and I absolutely admire your work!"
+* "Very nice OS with a helpful community. Thanks!"
+* "its pretty cozy and i like how lightweight it is."
+* "Keep up the good work."
+* "Ship it!"
+* "Keep up the great work!"
+* "You did a great job!"
+* "You guys do a great job, add better graphics support (from FreeBSD?)"
+
+
+**Here are some of the random critical opinions**<br/>
+
+* "Another problem is the lack remembering NFS mounts and not mounting NFS submounts. Also the BTRFS crashes the OS which makes sharing user files with Linux not possible."
+* "I still think PM was a mistake, at least in this implementation because it caused too many divisions in the community."
+* "Multi-monitor support is a must"
+* "Need to start anticipating multi user support"
+* "I'm a classroom teacher, so a single user OS is of little use for me on lab machines. I would love to use it on Raspberry Pi's, but it seems the Haiku community is not much interested in that.  Oh well."
+* "Haiku team should prioritize on releasing R1 as soon as possible. Everything else can wait. It's important in order to attract new developers."
+* "The Alt vs Ctrl issue is getting on my nerves, and I've used BeOS since 2001. Makes switching between Win/Linux/Haiku a pain in the ass everytime Haiku does it differently."
+* "No working video playback in webpositive and otter browsers"
+
+
+## Overall Summary
+
+Looking at the big picture above, several things stand out.
+
+1) We need to put a strong focus on continuing to improve WebPositive. 
+2) We need to work on a roadmap of how Haiku could present Linux DRM/KMS compatibility API's to Mesa. We really can't make any accelerated graphics progress without this.
+
+I personally think #1 is a lot more possible in the short term.<br/>
+ #2 is going to be a *LOT* of work in the medium / long term.
```

Comment:
Starting a sentence with `#2` doesn't seem correct to me. Maybe "But #2" or "On the other hand #2". But this doesn't need to be a blocker.